test/model/message/PersistentHarvestingDelegationMessage.spec.ts

```diff
@@ -89,16 +93,17 @@ describe('PersistentHarvestingDelegationMessage', () => {
             signedTransaction.payload.substring(322, signedTransaction.payload.length),
         );
         const plainMessage = PersistentHarvestingDelegationMessage.decrypt(encryptMessage, recipient.privateKey);
-        expect(plainMessage).to.be.equal(delegatedPrivateKey);
+        expect(plainMessage).to.be.equal(signingPrivateKey + vrfPrivateKey);
```

Comment:
plainMessage should be a pair of the 2 private keys already split

Reply:
yeah. not expecting normal user to decrypt the message, only the node owner, I guess the combined key string should be okay

Reply:
Fair enough